### PR TITLE
Feature/mock data

### PR DESCRIPTION
**Pull request by PaigeVannelli**
## Is this a fix or a feature?
Feature - add mock data file 

## What is the change?
Add mock data to use for component setup before getting data from API 

## What does it fix?
n/a

## Where should the reviewer start?
Review the mockData file in src 

## How should this be tested?
n/a

## Relevant ScreenShots
n/a

### CODE REVIEW

File: src/mockData.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  },
+  route: {
+    origin: '2199 S University Blvd, Denver, CO 80208',
+    destination: '1850 Table Mesa Dr, Boulder, CO 80305',
+    distanceFromOrigin: '0.5',
+    distanceFromDestination: '1.2',
+    time: '08:15',
+    days: ['monday', 'tuesday', 'wednesday', 'thursday', 'friday']
+  }
+}
```
Comment (alexthompson207) [inline src/mockData.js]: This is some really solid test data, good work!

**Review by alexthompson207 — APPROVED** [review]
T

**Review by itsnameissteven — APPROVED** [review]
looks good to me. Ill try and add some mock data as well on my next push

**Review by alexthompson207 — APPROVED** [review]
Nice catch in exporting the test data

**Pull request merged**